packages/text-annotator/test/index.html

```diff
@@ -154,12 +154,16 @@ <h1>Homer: The Odyssey</h1>
     </div>
 
     <script type="module">
-      import { createTextAnnotator } from '../src/index.ts';
+      import { createTextAnnotator, W3CTextFormat } from '../src/index.ts';
 
-      window.onload = async function() {
-        var r = createTextAnnotator(document.getElementById('content'));
+      window.onload = async () => {
+        const contentContainer = document.getElementById('content');
 
-        r.loadAnnotations('annotations.json');
+        const r = createTextAnnotator(contentContainer, {
+          adapter: W3CTextFormat('https://www.gutenberg.org', contentContainer)
+        });
+
+        void r.loadAnnotations('annotations.w3c.json');
```

Comment:
`annotations.json` file was removed way back in this PR - https://github.com/recogito/text-annotator-js/pull/16